### PR TITLE
Tile component

### PR DESCRIPTION
**Pull request by ReenaOmana**
Please review the component

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/.content.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="tile component for product listing page"
+    jcr:primaryType="cq:Component"
+    jcr:title="Tile"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/.content.xml]: @ReenaOmana : Component name will be Product Listing Tile

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/_cq_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                        fieldDescription="enter the brand here"
+                        fieldLabel="Brand"
+                        name="./brand"/>
+                    <tag
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
+                        fieldDescription="enter the tag here"
+                        fieldLabel="tag"
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/_cq_dialog/.content.xml]: @ReenaOmana : field Label to be Data-Tag Value

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+tile component
+<div class='product-listing-tile' data-tag=${properties.tag}>
+<br>
+<div data-sly-resource="${'img' @ resourceType='hclecomm/components/content/image', decoration=true}"> </div>
+<br>
+<p style="text-align:center;">${properties.brand} </p>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html]: @ReenaOmana : why doe we have inline style here ?

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html.dir/.content.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html.dir/.content.xml]: @ReenaOmana : why has the .dir file been checked in ?

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
@ReenaOmana : Pl incorporate comments

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/.content.xml (hunk 1 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/.content.xml]: Changes done

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/_cq_dialog/.content.xml (hunk 2 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/_cq_dialog/.content.xml]: Field label is changed

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html (hunk 3 above)
Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tile/tile.html]: This is removed

**Pull request merged**